docs/resources/project.md

```diff
@@ -28,7 +28,7 @@ An ASA construct that contains servers and is used to grant end users access to
 - `next_unix_uid` (Number) The UID to use when creating a new ASA Server User. Default value starts at 60001.
 - `rdp_session_recording` (Boolean) If 'true', enable remote desktop protocol (RDP) recording on all servers in the ASA Project.
 - `require_preauth_for_creds` (Boolean) If 'true', require preauthorization before an ASA User can retrieve credentials to sign in.
-- `ssh_certificate_type` (String) The SSH certificate type used by access requests. Options include: `CERT_TYPE_ED25519_01`, `CERT_TYPE_ECDSA_521_01`, `CERT_TYPE_ECDSA_384_01`, `CERT_TYPE_ECDSA_256_01` and `CERT_TYPE_RSA_01`. `CERT_TYPE_RSA_01` is a deprecated key algorithm type. This option should only be used to connect to legacy systems that cannot use newer SSH versions. If you do need to use `CERT_TYPE_RSA_01`, it is recommended to connect via a gateway with traffic forwarding. Otherwise, please use a more current key algorithm. If left unspecified, `CERT_TYPE_ED25519_01` is used by default.
+- `ssh_certificate_type` (String) The SSH certificate type used by access requests. Options include: [`CERT_TYPE_ECDSA_521_01`, `CERT_TYPE_ECDSA_384_01`, `CERT_TYPE_ECDSA_256_01`, `CERT_TYPE_ED25519_01`, `CERT_TYPE_RSA_SHA2_512_01`, `CERT_TYPE_RSA_SHA2_256_01`, `CERT_TYPE_RSA_01`]. `CERT_TYPE_RSA_01` is a deprecated key algorithm type. This option should only be used to connect to legacy systems that cannot use newer SSH versions. If you do need to use `CERT_TYPE_RSA_01`, it is recommended to connect via a gateway with traffic forwarding. Otherwise, please use a more current key algorithm. If left unspecified, `CERT_TYPE_ED25519_01` is used by default.
```

Comment:
Nit, but since CERT_TYPE_ED25519_01 is our default it feels like it would be better if this was the first item in this list.